news/2023/2023-01-31-skinning-poll-2023.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "osu! 2023 Skinning Poll"
+date: 2023-01-31 17:00:00 +0000
+---
+
+Are you an avid skin creator who makes skins from scratch, or someone who makes remixed skins for fun? Maybe you are just a person who likes collecting skins and exploring what the community has to offer? Either way, we want your feedback on everything skinning related!
+
+![](https://assets.ppy.sh/media/generic-header.png)
+
+With skinning continuously evolving, we created a survey through which the community can express their opinion on the current state of the skinning scene and help us better understand the community's expectations and needs.
+
+The link below will lead you to the survey page, while most questions are not mandatory to answer, the more feedback we receive, the easier it will be for us to make changes you want to see happen come to life.
+
+- [osu! 2023 Skinning Poll](https://docs.google.com/forms/d/e/1FAIpQLScEtYo-uxu-uSS7S3C_y7DV-EZWY47dfg9-YpUuThu8belCQw/viewform)
+
+The survey should only a couple of minutes to complete, so be sure to pass the news along to any of your friends that might be interested in skinning! Your opinion is extremely valuable to us.
+
+For those who wish to further engage with the skinning community, please feel free to take a look at the [skinning tutorial](https://skinship.xyz/tutorial/introduction) or check out the [skinning forum](https://osu.ppy.sh/community/forums/15)!
+
+There is currently no set date for announcing the survey results, but when the time comes it will be announced via the [skinship Discord server](https:/discord.skinship.xyz), and a write-up of the results will be released on the skinning forums.
```

Comment:
but not as a news post? dunno how to feel about that

Reply:
@RockRoller01 will need to comment on that

Reply:
Ive talked about this with eph and forum post was actually his suggestion, which I fully agree with. Results probably are not newspost worthy, already is going to be hard to come up with anything valuable for the public to see

Reply:
if Wala is satisfied with that answer I will resolve this

Reply:
seems fine then, but the skinning forum can be linked (the place where you'd see the results), and to me it would be even better to have a set date

Reply:
Can this be done with some sort of forum thread that can be linked even though its not live yet? So it would go live once the results are published, but the link would already be on the news post?

Reply:
> and to me it would be even better to have a set date
anywhere between the end date and 2 months later. I am simply speaking unable to commit to any specific date on this. No clue how long this will take to go trough, nor do I know yet how much time I will have.
> skinning forum can be linked
yea good catch, its arleady mentioned so converting it to a link shouldn't be hard

Reply:
> Can this be done with some sort of forum thread that can be linked even though its not live yet? So it would go live once the results are published, but the link would already be on the news post?

that sounds a bit weird, i more so meant just linking to the skinning forum

Reply:
This can be marked as resolved now, handed in another suggestion to fix this, since i couldn't do it here, because this was outdated